src/Server/RestRouterHandlers/RestRouterFactory.h

```diff
@@ -120,6 +121,13 @@ class RestRouterFactory final
             [](ContextPtr query_context) { /// STYLE_CHECK_ALLOW_BRACE_SAME_LINE_LAMBDA
                 return std::make_shared<DB::RestStatusHandler>(query_context);
             });
+
+        factory.registerRouterHandler(
+            "/dae/v1/search",
```

Comment:
I was wondering if we can provide 2 output format : 
1. JSONCompact which is very frontend / user friendly
2. JSONCompactEachRowWithNamesAndTypes which may be more efficient and machine friendly (dump data / load data etc) and data can be sent back to user as soon as possible.